<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HotSpot</class>
 <widget class="QWidget" name="HotSpot">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>418</width>
    <height>536</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>418</width>
    <height>536</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>418</width>
    <height>536</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="widget_Photo" native="true">
     <property name="minimumSize">
      <size>
       <width>400</width>
       <height>300</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>400</width>
       <height>300</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_changePhoto">
       <property name="text">
        <string>更改图片</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTextEdit" name="textEdit_Description">
     <property name="minimumSize">
      <size>
       <width>400</width>
       <height>130</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>400</width>
       <height>130</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_position">
       <property name="text">
        <string>坐标</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_deleteHotSpot">
       <property name="maximumSize">
        <size>
         <width>75</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>删除热点</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <zorder>widget_Photo</zorder>
  <zorder>textEdit_Description</zorder>
  <zorder>label_position</zorder>
  <zorder>horizontalSpacer</zorder>
  <zorder>pushButton_changePhoto</zorder>
  <zorder>pushButton_deleteHotSpot</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
